### ISSUE TITLE
Enhancements to the recommended/required plugins page

### ISSUE THREAD

**Issue by vpratfr** [thomasgriffin/TGM-Plugin-Activation#278]
As a theme author, I would like to be able to advise on some recommended plugins. There may be several that I could recommend but I would like to provide alternatives and maybe give a little opinion/thoughts about each alternative.

For instance, in my themes, I would suggest plugins in the following categories:

1. Google Analytics
  a. Plugin A - provides quick and easy integration and the bare minimum you'd need. 
  a. Plugin B - a feature-full analytics integration, dashboard widgets and all. 
1. Security
1. SEO
1. etc.

And so on. In this case, the recommended plugins list would be long and arranging it with sections and giving comments about each plugin would help the end user decide which one he would like to install.

**Comment by jrfnl** [comment]
Partially duplicate of issue #259 (Opinion/thoughts).

**Comment by vpratfr** [comment]
Yep, I had not seen that one, I feel grouping would help a lot when presenting a lot of recommendations.

I would be glad to contribute to this one once #265 is done, I feel that one is the first issue to address. Do not hesitate to poke me !

**Comment by jrfnl** [comment]
@vpratfr I've had a bit of a think about the grouping and to be honest, I don't think it's feasible at all.
The thing is that several themes/plugins can all use TGMPA and can recommend/require the same plugin. They could each tag it with a different section name which would result in a bit of a mess (duplicate plugin mentions or "fights" between theme/plugin authors about which section a plugin should go in), quite apart from the whole spelling thing which opens a whole other can of worms... 

What about allowing author to tag plugins with some keywords instead (comma delimited list) ? Those could be shown with the plugin and can be de-duped quite easily and slight spelling differences won't cause real issues.

**Comment by jrfnl** [comment]
@vpratfr Oh and I'd hold off on contributing for the moment until I've set up an alpha version of v3 as there are going to be some significant changes.
All the same -  it would be **awesome** if you'd be willing to beta test once the alpha/beta version of v3 is available!

**Comment by vpratfr** [comment]
i think tagging is a good alternative and would be a great improvement. I
had not anticipated the "shared" usage of tgm.

This along with a tag cloud to ajaxaly hide the table entries not
corresponding to the selected tag would be nice.
